spec/features/editing_content_settings/political_spec.rb

```diff
@@ -0,0 +1,82 @@
+# frozen_string_literal: true
+
+RSpec.feature "Political content" do
```

Comment:
I'd suggest naming this history mode since that's a bit more common way of referring to the feature than political.